src/components/SoraCard/steps/ToSDialog.vue

```diff
@@ -1,12 +1,23 @@
 <template>
-  <dialog-base :visible.sync="isVisible" class="terms-of-service-dialog" title="Terms & Conditions">
-    <div v-loading="loading" class="tos__section">
+  <dialog-base :visible.sync="isVisible" class="terms-of-service-dialog" :title="title">
+    <div v-if="!srcLink" class="sora-card__excuse">
+      Unfortunately, at this moment we’re not able to support the following countries:
+    </div>
+    <div v-if="srcLink" v-loading="loading" class="tos__section">
       <iframe @load="onIFrameLoad" :src="srcLink" width="100%" height="600px" frameborder="0"></iframe>
     </div>
+    <div v-else class="tos__section">
+      <ul class="sora-card__unsupported-countries">
+        <li v-for="(country, index) in unsupportedCountries" :key="country">
```

Comment:
... `v-for="[code, value] in unsupportedCountries.entries()" :key="code">`

... `{{ countryCodeEmoji[code] }}` ...